policy-engine/src/main.rs

```diff
@@ -44,10 +44,11 @@ fn main() -> Result<(), Error> {
         .init();
 
     // Metrics service.
+    let metrics_path = opts.path_namespace.clone() + "/metrics";
     server::new(move || {
         App::new()
             .middleware(Logger::default())
-            .route("/metrics", Method::GET, metrics::serve)
+            .route(&metrics_path, Method::GET, metrics::serve)
```

Comment:
This is on a different port, and not be exposed as a public API. I think it shouldn't be prefix.